src/teamplay.c

```diff
@@ -875,9 +956,20 @@ static void TeamplayAreaSecure(gedict_t *client)
 		return;
 	}
 
+	if (HAVE_POWERUP(client))
+	{
+		strlcpy(buffer, PowerupText(client), sizeof(buffer));
+		strlcat(buffer, " ", sizeof(buffer));
+	}
+	if (isCTF() && (HAVE_RUNE(client) || HAVE_RUNE(client)))
+	{
+		strlcpy(buffer, CTFItemText(client), sizeof(buffer));
```

Comment:
Is it intentional that you overwrite the buffer here and everywhere else (can have powerup message in it).